src/app/pages/choose-account/choose-account.tsx

```diff
@@ -34,10 +38,14 @@ export const ChooseAccount = memo(() => {
           {url && <RequesterFlag requester={url.toString()} />}
           <LogomarkIcon width="248px" height="58px" />
           <Stack gap="space.04">
-            <styled.h1 textStyle="heading.05">Choose an account to connect</styled.h1>
+            <styled.h1 textStyle="heading.05">
```

Comment:
I improvised and added a message here: 

![Screenshot 2024-04-15 at 10 31 02](https://github.com/leather-wallet/extension/assets/2938440/39602fc7-6449-4608-a991-f238a1cf18df)

When using Ledger, you an connect BTC and not STX. If this is the case and you try and connect your wallet with Gamma / some service you got a popup saying 'Choose an account to connect' but no accounts are listed. 

I initially changed it to the below, however I didn't think it made sense to say 'Choose account' then 'No Accounts'
![Screenshot 2024-04-15 at 10 22 55](https://github.com/leather-wallet/extension/assets/2938440/9699b61e-cd22-4ca5-9372-1176fd2dc928)